darc/darc.go

```diff
@@ -1447,6 +1500,22 @@ func (kcs SignerX509EC) Sign(msg []byte) ([]byte, error) {
 	return nil, errors.New("not yet implemented")
 }
 
+// new signer creates a signer only with a public key used to verify signatures
+func NewSignerECDSA(public ecdsa.PublicKey) Signer {
+	if public.X == nil {
+		return Signer{}
+	}
+	return Signer{ECDSA: &SignerECDSA{
+		PublicKey: public,
+	}}
+}
+
+//TODO
```

Comment:
One way to implement sign would be to give the necessary configuration when calling `NewSignerECDSA`, and then having `Sign` call the MPC backend. But that would introduce a dependency on the MPC code that we probably don't want to have in here.

```suggestion
// Sign cannot be implemented, as only the MPC can sign a message.
```